src/_incydr_sdk/watchlists/models/responses.py

```diff
@@ -149,14 +189,34 @@ class IncludedUsersList(ResponseModel):
     A model representing a list of users included on a watchlist.
     Included users are those that have been individually included on that list.
 
-    * **included_users**: `List[WatchlistUser]` - The list of included users.
-    * **total_count**: `int` - The total count of all included users.
+    * **included_users**: `List[WatchlistUser]` - The list of included users or actors.
```

Comment:
we can probably just say `actors` instead of `users or actors` for docstrings?

Reply:
Ah, that's left over from when I was trying to make the model work for both cases.